src/agent.rs

```diff
@@ -423,6 +424,8 @@ impl<AGN: Agent> RemoteAgent<AGN> {
 
 thread_local! {
     static REMOTE_AGENTS_POOL: RefCell<AnyMap> = RefCell::new(AnyMap::new());
+    static DEDICATED_WORKERS_LOADED: RefCell<HashSet<&'static str>> = RefCell::new(HashSet::new());
```

Comment:
I'm not a fan of using the `name_of_resource` as the index. How about using [TypeId](https://doc.rust-lang.org/std/any/struct.TypeId.html)? (_this is actually what `REMOTE_AGENTS_POOL` is using too under the hood in `AnyMap`_)

Reply:
Also, I think we should stick with common naming, how about calling this `REMOTE_AGENTS_LOADED`?

Reply:
NP will rename it

Reply:
@serzhiio I already did, please pull latest changes locally 😄